backend/src/middleware/permissionsMiddleware.js

```diff
@@ -133,7 +137,7 @@ export default shield(
       CreateComment: isAuthenticated,
       UpdateComment: isAuthor,
       DeleteComment: isAuthor,
-      DeleteUser: isDeletingOwnAccount,
+      DeleteUser: or(isDeletingOwnAccount, isAdmin),
```

Comment:
This is right done!
But may be not enough. That’s why you have added the odd code above. ☝🏼